source/administration/changelog.md

```diff
@@ -4,6 +4,149 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.18 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
+
+**Release day: 2019-12-16**
+
+Mattermost v5.18.0 contains low to high level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Compatibility
+ 
+### Breaking Changes
+ - Breaking change for mark posts as unread **XXX**
```

Comment:
Don't need a breaking change for mark as unread since it's not breaking anything, it's just something to be aware of while upgrading. We should add this to the upgrade notes instead: https://docs.google.com/document/d/1Dtfem7No4Ws6NT0icKPNYZCtpH954N9HW2jN60-5ptE/edit#bookmark=id.6s51w98icjz3